content/en/synthetics/guide/synthetic-test-monitors.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Understanding Synthetic Tests and Alerting Monitors
+kind: guide
+description: Learn about Synthetic monitors created with your Synthetic tests. 
+further_reading:
+- link: "/monitors/manage/"
+  tag: "Documentation"
+  text: "Manage Monitors"
+- link: "/synthetics/metrics/"
+  tag: "Documentation"
+  text: "Synthetic Monitoring Metrics"
+---
+
+## Overview
+
+When you create a Synthetic test, Datadog automatically creates an associated monitor. The monitor's alerting conditions in your test configuration impact how you are notified when a Synthetic test moves into an alerting state.
+
+{{< img src="synthetics/guide/synthetics_test_monitors/synthetic_test_monitor.png" alt="Synthetic test monitor" style="width:100%;">}}
+
+This guide describes how Synthetic test monitors behave, how to use a monitor's conditional variables in a Synthetic test configuration, and how to set downtimes and mute monitors.
+
+## Create a Synthetic test monitor
+
+<div class="alert alert-info">Alerting monitors created by Synthetic tests are not a <a href="/monitors/create/">monitor type</a>. You cannot create or import a Synthetic test monitor.</div>
```

Comment:
_Technically_ they are a monitor type, they are just very unique/we don't surface them within the monitor creation workflow (which is why you cannot create/import them).

Reply:
+1, users can filter on `type:synthetics` on the `Manage Monitors` page.